vignettes/EpiNow2.Rmd

```diff
@@ -87,6 +87,22 @@ example_incubation_period
 #>         0.069
 ```
 
+Users can also pass a non-parametric delay distribution vector using the `NonParametric` option 
+for both the generation interval and reporting delays. It is important to note that if doing so,
+both delay distributions are 0-indexed, meaning the first element corresponds to the probability mass
+at day 0 of an individual's infection. Because the discretised renewal equation doesn't support mass on day 0, the generation interval should be passed in as a 0-indexed vector with a mass of zero on day 0. 
+If this is not the case, a warning will indicate that the vector is being left-truncated and renormalized.
+
+```r
+example_non_parametric_gi <-  NonParametric(pmf = c(0, 0.3, 0.5, 0.2))
+
+example_non_parametric_delay <- NonParametric(pmf = c(0.01, 0.1, 0.5, 0.3, 0.09))
+```
+These distributions are passed to downstream functions in the same way that the 
+parametric distributions are. The difference in the `NonParametric` case is that 
+these will be treated as fixed rather than estimated with the specified parametric priors. 
+
```

Comment:
```suggestion
parametric distributions are. 
```

This is not strictly true unless (normal) priors are specified for the parameters of the parametric distributions. If passing a parametric distributions with fixed parameters (e.g. `Lognormal(meanlog = 1, sdlog = 1)`) these will internally be converted to a nonparametric distribution and treated as fixed.